Yelp/nrtsearch
# Support epoch_millis in DateTimeField

### Changes 
Support `epoch_millis` as DATE_TIME field format

### Testing
Unit tests:
* range query for epoch_millis DATE_TIME
* adding docs with epoch_millis DATE_TIME field

Manual testing:
* Registered following fields
```
'{                                                      
   "indexName":"testIdx",
   "field":[
      {
         "name":"id",
         "type":"ATOM",
         "storeDocValues":true
      },
      {
         "name":"title",
         "type":"TEXT",
         "search":true,
         "store":true,
         "tokenize":true
      },
                        {
        "name": "timestamp",
        "type": "DATE_TIME",
        "storeDocValues": true,
        "dateTimeFormat": "epoch_millis"
      }
   ]
}
```
* Index following csv document
```
id,title,timestamp
0,first vendor,1611678698815
1,second vendor,1611678827275
```
* Query the index, timestamp is added as expected
```
{"diagnostics":{"parsedQuery":"*:*","rewrittenQuery":"*:*","firstPassSearchTimeMs":3.678646,"getFieldsTimeMs":15.7659},"totalHits":{"value":"2"},"hits":[{"score":1,"fields":{"id":{"fieldValue":[{"t$xtValue":"0"}]},"timestamp":{"fieldValue":[{"longValue":"1611678698815"}]},"title":{"fieldValue":[{"textValue":"first vendor"}]}}},{"luceneDocId":1,"score":1,"fields":{"id":{"fieldValue":[{"textValue":"1"}]},"timestamp":{"fieldValue":[{"longValue":"1611678827275"}]},"title":{"fieldValue":[{"textValue":"second vendor"}]}}}],"searchState":{}}%                                                     
```
